pyproject.toml

```diff
@@ -0,0 +1,73 @@
+[tool.poetry]
+name = "jira-bugzilla-integration"
+version = "0.1.0"
+description = "jira-bugzilla-integration"
+authors = ["@mozilla/jbi-core"]
+license = "MPL"
+
+[tool.poetry.dependencies]
+python = ">=3.8, <3.11"
+
+[tool.poetry.dev-dependencies]
+pre-commit = "^2.17.0"
+black = "^22.1.0"
+isort = {extras = ["pyproject"], version = "^4.3.21"}
+coverage = {extras = ["toml"], version = "^5.3"}
+mypy = "^0.910"
+detect-secrets = "^0.14.3"
+bandit = "^1.7.0"
+pylint = "^2.7.2"
+pylint-pytest = "^1.1.2"
+pytest = "^7.0.0"
+
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.pytest.ini_options]
+testpaths = [
+    "tests/unit",
+]
+
+[tool.pylint]
+    [tool.pylint.'MESSAGES CONTROL']
+    disable = [
+        "C0114", #missing-module-docstring
+        "C0116", #missing-function-docstring
```

Comment:
https://github.com/mozilla/jira-bugzilla-integration/issues/7

Made this an issue; I believe this can be resolved over time, and would be helpful for individuals looking to contribute to it as open source.